### ISSUE TITLE
error auto stop the server

### ISSUE THREAD

**Issue by Eightx3** [RonanPlugins/BetterRTP#215]
**Plugin Version**:
BetterRTP-3.6.13
https://mclo.gs/PuGpcUM
when i /betterrtp location in console show this error fix pls

**Comment by Eightx3** [comment]
** after mv regen world**

**Comment by ARR4NN** [comment]
Have you tried restarting the server after a mv regen?

**Comment by Eightx3** [comment]
yes restart it work but I don't need to restart server it resourceworld regen daily

**Comment by ARR4NN** [comment]
Try disable queue maybe

**Comment by Eightx3** [comment]
not work same error
